src/index.ts

```diff
@@ -533,6 +536,19 @@ export class Squid {
     );
     const gasMultiplier = Number(route.transactionRequest!.maxFeePerGas) || 1.3;
 
+    if (useBroadcast) {
+      const fee = calculateFee(
+        Math.trunc(estimatedGas * gasMultiplier),
+        GasPrice.fromString(route.transactionRequest!.gasPrice)
+      );
+
+      return (signer as SigningCosmWasmClient).signAndBroadcast(
+        signerAddress,
+        [fromAminoMsg],
+        fee,
+        ""
+      );
+    }
     return (signer as SigningCosmWasmClient).sign(
       signerAddress,
       [fromAminoMsg],
```

Comment:
The calculateFee method is called in If and also in Else (return)
So we're sure to use it, we can define it as a constant


```ts
const gasMultiplier = Number(route.transactionRequest!.maxFeePerGas) || 1.3;
const fee = calculateFee(
  Math.trunc(estimatedGas * gasMultiplier),
  GasPrice.fromString(route.transactionRequest!.gasPrice)
);

if (useBroadcast) {
  return (signer as SigningCosmWasmClient).signAndBroadcast(
    signerAddress,
    [fromAminoMsg],
    fee,
    ""
  );
}
return (signer as SigningCosmWasmClient).sign(
  signerAddress,
  [fromAminoMsg],
  fee,
  ""
);
```